docs/tutorials/deploy-vm/cloudinit.md

```diff
@@ -1,130 +1,20 @@
-| [Sysprep](#sysprep)         |       ✓      | [Guest OS Customization](https://vdc-download.vmware.com/vmwb-repository/dcr-public/c476b64b-c93c-4b21-9d76-be14da0148f9/04ca12ad-59b9-4e1c-8232-fd3d4276e52c/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.customization.Specification.html) (GOSC) |       |     ✓    | Microsoft Sysprep is used by VMware to customize Windows images on first-boot |
-| [vAppConfig](#vappconfig)   |       ✓      | Bespoke                       |   ✓   |         | For images with bespoke, bootstrap engines driven by vAppConfig properties |
-| [OvfEnv](#ovfenv)           | _deprecated_ | [Guest OS Customization](https://vdc-download.vmware.com/vmwb-repository/dcr-public/c476b64b-c93c-4b21-9d76-be14da0148f9/04ca12ad-59b9-4e1c-8232-fd3d4276e52c/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.customization.Specification.html) (GOSC) |   ✓   |         | A combination of GOSC and Cloud-Init user-data |
-| [ExtraConfig](#extraconfig) | _deprecated_ | GOSC                          |   ✓   |         | For images with bespoke, bootstrap engines that rely on Guest Info data |
-
-!!! note "`ConfigMap` or `Secret`"
-
-    The choice of a `ConfigMap` or `Secret` resource in no way impacts the choice or behavior of the selected bootstrap provider. When VM Operator was first released, the only way to store bootstrap data was via a `ConfigMap` resource. While this still works, it is not recommended as data stored in a `ConfigMap` is not encrypted at rest. Instead, it is recommended users switch to using `Secret` resources for storing bootstrap data. Aside from how the data is stored in etcd, the following two resources are effectively identical and serve the same purpose:
-
-    === "CloudConfig in a ConfigMap"
-
-        ``` yaml
-        apiVersion: v1
-        kind: ConfigMap
-        metadata:
-          name:      my-vm-bootstrap-data
-          namespace: my-namespace
-        data:
-          user-data: |
-            #cloud-config
-            users:
-            - default
-        ```
-
-    === "CloudConfig in a Secret"
-
-        ``` yaml
-        apiVersion: v1
-        kind: Secret
-        metadata:
-          name:      my-vm-bootstrap-data
-          namespace: my-namespace
-        stringData:
-          user-data: |
-            #cloud-config
-            users:
-            - default
-        ```
-### Supported
-
-#### Cloud-Init
-
-[Cloud-Init](https://cloudinit.readthedocs.io/en/latest/) is widely recognized as the de facto method for bootstrapping modern VM instances on hyperscalers, including Tanzu with VM Operator. For example, the following resources may be used to deploy a VM and bootstrap its guest with Cloud-Init to:
+## Purpose
+As a DevOps user, you can bootstrap the guest inside a VM using the Cloud-Init to:
```

Comment:
Not your change, but I don't think this is the exhaustive list of capabilities unlocked by cloud init user-data?  Maybe change the language to reflect that these are just an example of a few things that users can do.